src/tools/updater/main.cpp

```diff
@@ -93,7 +93,12 @@ class MyStudyFinder final : public Data::StudyFinder
                     study->parameters.readonly = true;
 
                 logs.info() << "Saving...";
-                study->saveToFolder(folder);
+
```

Comment:
Normally, we should add this to the **src/libs/antares/study/cleaner/cleaner.cpp**, which is supposed to clean any study from unsuitable files.

Reply:
You are right. I've looked up cleaner.cpp file in details, and apparently after upgrading and saving our new files, we will delete deprecated files with clean command, so I don't think that we need automatic deletion after upgrade. With that been said, this PR shouldn't be merged.